### ISSUE TITLE
[Question] can this support multimarkdown 6?

### ISSUE THREAD

**Issue by simkimsia** [tgrosinger/advanced-tables-obsidian#59]
reason is because i was hoping to have merged cells in the table.

See https://fletcher.github.io/MultiMarkdown-6/syntax/tables.html for example of merged cells

**Comment by tgrosinger** [comment]
@simkimsia, thank you very much for the kind words and support.

Unfortunately advanced-tables does not support multimarkdown. There are a couple of limitations which are equally important: 

- advanced-tables does not have any effect on how tables are rendered in Obsidian. It only operates in edit mode. Currently the API does not expose enough information for a plugin to add support for those tables. I believe the information necessary is planned, however there is no estimate on when that will come. This problem could also be worked around if Obsidian added support for MMD natively. Perhaps something to post in the [feature request forum](https://forum.obsidian.md/c/feature-requests/8)?
- The second issue is that the library powering advanced-tables ([md-advanced-tables](https://github.com/tgrosinger/md-advanced-tables)) does not yet support MMD tables. This problem is something I am able to do something about, however I probably will not put time into that until the first issue is resolved.

So shorter answer: I hope to do that someday! But waiting on support from Obsidian first.

**Comment by AidenLx** [comment]
Good News! I've written a plugin that allows MultiMarkdown table syntax to work in Obsidian, maybe you can consider first allow the `tx`-tagged code block to be ignored so that Advanced Table won't break the MMD tables before MMD is fully supported?

Link to my plugin: <https://github.com/alx-plugins/table-extended>

**Comment by tgrosinger** [comment]
@AidenLx how does Advanced Tables break the MMD tables?

**Comment by AidenLx** [comment]
The auto-formatting also applies to the table within the code block, so, for example, when hitting enter, the gaps between divide mark `|` is filled with blanks, and the header, if not present, will be added. These would break `colspan` and intentionally omitted table header

It could be better illustate in this demo: 
```tx
| ------------ | :-----------: | -----------: | 
|Content       |          *Long Cell*        || 
```
The auto-formatting would also affect multiple table headers, illustrated here: 
```tx
|             |          Grouping           || 
|First Header  | Second Header | Third Header | 
| ------------ | :-----------: | -----------: | 
|Content       |          *Long Cell*        || 
```

**Comment by AidenLx** [comment]
It's been a while since then, any update on the possible fix?

**Comment by tgrosinger** [comment]
No update. I have not had a chance to put much time into Obsidian plugins recently.

**Comment by dummifiedme** [comment]
Any update on this one? :crossed_fingers:

**Comment by tgrosinger** [comment]
I'm going to split this into two tickets. Supporting MMD is a large task that I don't anticipate adding in the near future, however ignoring those tables is smaller and could be done sooner.

**Comment by Type-IIx** [comment]
Hrm, only a half century since this issue was first raised. Maybe someone else – even myself – will take a shot?
